Composer/packages/electron-server/src/auth/oneAuthService.ts

```diff
@@ -34,6 +34,27 @@ type GetTenantsResult = {
   value: AzureTenant[];
 };
 
+enum Status {
```

Comment:
Where do these values come from? Maybe add a comment explaining where to find this list?

Reply:
So we have them in our copied version of OneAuth's  `d.ts` file ([Source here](https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/packages/electron-server/src/auth/oneauth.d.ts)). However, the problem is that, at run time, the `OneAuth` object that we initialize does not actually have these enums defined as properties on the object even though it is in the package's definition file.

So I extracted those enums out into this file, but I can add a comment pointing to the original definition file for reference.